docs/topics/assembly_creating-a-basic-wildfly-swarm-application.adoc

```diff
@@ -8,6 +8,18 @@ In addition to xref:mission-rest-http-wf-swarm[using a booster], you can create
 
 include::thorntail/docs/howto/writing-an-application-from-scratch/index.adoc[leveloffset=+1]
 
+:parameter-runtime-name: {Thorntail}
+:parameter-group-name-runtime-plugin: io.thorntail
+:parameter-artifact-name-runtime-plugin: thorntail-maven-plugin
+:parameter-version-runtime-plugin: ${thorntail-maven-plugin.version}
```

Comment:
I don't know about other runtimes, but in case of Thorntail, the Maven plugin version is identical to the Thorntail version itself. So this should be something like `${version.thorntail}` (I personally prefer the `version` string to be a prefix, not a suffix :-) ).

Reply:
Same thing for the Spring Boot plugin - the version of the plugin is the same as the version of Spring Boot you use

Reply:
Not in Vert.x case. The version of the plugin is different as the version of Vert.x.